packages/thumbprint-scss/__snapshots__/test.js.snap

```diff
@@ -2,7 +2,6 @@
 
 exports[`compiles correctly 1`] = `
 ".tp-alert {
-  display: -webkit-flex;
```

Comment:
Is this an issue?

Reply:
Brought this up on Slack in `#front-end-infra` to see who has concerns.

I confirmed that not having flex prefixes is the correct behavior for our `browserslist` config. We happened to still be adding them because our packages were old so the browser usage data was stale.

Reply:
Got a 👍to kill the prefixes.